src/rules/html/warn-on-remote-script.js

```diff
@@ -0,0 +1,25 @@
+import { VALIDATION_WARNING } from 'const';
+import { isStrictRelativeUrl } from 'schema/formats';
+import * as messages from 'messages';
+
+
+export function warnOnRemoteScript($, filename) {
+  return new Promise((resolve) => {
+    var linterMessages = [];
+
+    $('script').each((i, element) => {
+      const src = $(element).attr('src');
+
+      if (src !== undefined && !isStrictRelativeUrl(src)) {
+        linterMessages.push(
+          Object.assign({}, messages.REMOTE_SCRIPT, {
+            type: VALIDATION_WARNING,
```

Comment:
That's a policy change that will need to be discussed separately from this issue. The risk factors and behavior of the linter were agreed upon months ago when we covered this in Toronto. I'd rather not make last-minute changes without discussion.